backend/core/src/seeds/listings/listing-detroit-treymore.ts

```diff
@@ -154,6 +155,34 @@ export class ListingTreymoreSeed extends ListingDefaultSeed {
       preferences: [],
     }
 
+    const treymoreUnitsSummaryToBeCreated: UnitsSummaryCreateDto[] = []
```

Comment:
Likewise can you put the actual available units in here with the correct monthly rent ranges? See rentlinx link.